### DISCUSSION TITLE
Open SSH is missing its openssl dependancy?

### DISCUSSION THREAD

**Discussion by Merith-TK** [termux/termux-packages#18884] [General]
On three different devices, I have recently gotten the error when installing openssh that `libssl` was missing and that openssh could not be setup (fails on generating hostkeys)

installing the `openssl` package, and then re-installing the `openssh` package fixes this issue
Devices are 
- Rooted Moto G Power (2021)
- Galaxy XCover Pro
- Galaxy Watch 4 (classic black)

**Comment by Merith-TK** [comment]
Update, Looked at the packages build.sh file, and it does seem like it has openssl listed as a dependancy, but it never installs it...

**Comment by sylirre** [comment]
Usually `openssl` becomes "missing" when user forgets that `pkg upgrade` has to be run on fresh Termux installation.

This is why *installing the openssl package, and then re-installing the openssh package fixes this issue*.
